src/index.ts

```diff
@@ -116,13 +116,44 @@ if (Config.nosecurity) {
 }
 
 server.get('/', async (req: AuthenticatedRequest, res) => {
-    let html = `Hello! This is the FRC Team ${Config.teamNumber || 5940} scouting website.`;
+    let html = ``;
 
     if (req.oidc?.isAuthenticated && req.oidc.user) {
-        html += `<p>You are logged in as ${(req.oidc.user as any).name || 'an unknown user'}. `;
-        html += `<a href="/logout">Log out</a></p>`;
+      html += `<html>
+        <head>
+          <title>Home</title>
+          <link rel="stylesheet" href="css/home.css">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        </head>
+        <body><div class="logged-in-content">
+          <div class="text-group">
+            Hello! This is the FRC Team ${Config.teamNumber || 5940} scouting website.
+            <p>You are logged in as You are logged in as ${(req.oidc.user as any).name
```

Comment:
```suggestion
            <p>You are logged in as ${(req.oidc.user as any).name || 'an unknown user'}. 
```
lmao, idk what happened here